README.md

```diff
@@ -16,49 +16,47 @@ Inferno is an open-source collection of fragment shaders designed for SwiftUI ap
 
 If you’re already comfortable with shaders then please download one or more that interest you and get going. If not, most of the remainder of this README acts as a primer for using shaders in SwiftUI.
 
-
-* [See it in action](#see-it-in-action)
-* [How to use Inferno in your project](#how-to-use-inferno-in-your-project)
-* [What are shaders?](#what-are-shaders)
-* [How are shaders written?](#how-are-shaders-written)
-* [Sending values to shaders](#sending-values-to-shaders)
-* [Reading the shader code](#reading-the-shader-code)
-* [Shaders included in Inferno](#shaders-included-in-inferno)
-    * [Animated Gradient Fill](#animated-gradient-fill)
-    * [Checkerboard](#checkerboard)
-    * [Circle Wave](#circle-wave)
-    * [Color Planes](#color-lanes)
-    * [Emboss](#emboss)
-    * [Gradient Fill](#gradient-fill)
-    * [Infrared](#infrared)
-    * [Interlace](#interlace)
-    * [Invert Alpha](#invert-alpha)
-    * [Light Grid](#light-grid)
-    * [Passthrough](#passthrough)
-    * [Rainbow Noise](#rainbow-noise)
-    * [Recolor](#recolor)
-    * [Relative Wave](#relative-wave)
-    * [Simple Loupe](#simple-loupe)
-    * [Sinebow](#sinebow)
-    * [Warping Loupe](#warping-loupe)
-    * [Water](#water)
-    * [Wave](#wave)
-    * [White Noise](#white-noise)
-* [Transitions included in Inferno](#transitions-included-in-inferno)
-    * [Circle](#circle)
-    * [Circle Wave](#circle-wave)
-    * [Crosswarp LTR](#crosswarp-ltr)
-    * [Crosswarp RTL](#crosswarp-rtl)
-    * [Diamond](#diamond)
-    * [Diamond Wave](#diamond-wave)
-    * [Pixellate](#pixellate)
-    * [Radial](#radial)
-    * [Swirl](#swirl)
-* [Inferno Sandbox](#inferno-sandbox)
-* [Contributing](#contributing)
-* [License](#license)
-* [Where to learn more](#where-to-learn-more)
```

Comment:
These changes are necessary. `*`s function the same as `-`s do.

Reply:
Sorry, I’m slightly confused. Are you simply noting that using ‘-’ is acceptable, or do you want me to change it?